tests/playwright-test/to-have-screenshot.spec.ts

```diff
@@ -922,6 +943,9 @@ test('should update expectations with retries', async ({ runInlineTest }, testIn
 });
 
 function playwrightConfig(obj: any) {
+  obj.expect ??= {};
+  obj.expect.toHaveScreenshot ??= {};
+  obj.expect.toHaveScreenshot._useScreenshotsDir ??= true;
```

Comment:
I guess this flag is temporary, just for tests?

Reply:
Yes, unless we decide to release screenshotsDir somehow later in the future.